extension/parquet/column_reader.cpp

```diff
@@ -1017,6 +1017,10 @@ ColumnReader *StructColumnReader::GetChildReader(idx_t child_idx) {
 void StructColumnReader::InitializeRead(idx_t row_group_idx_p, const vector<ColumnChunk> &columns,
                                         TProtocol &protocol_p) {
 	for (auto &child : child_readers) {
+		// make sure that child is not a nullptr
```

Comment:
Maybe a comment like `This column was skipped by projection pushdown` explains why this child is null

Reply:
Good point. Changed it now